src/Enterspeed.Cli/Configuration/Settings.cs

```diff
@@ -3,5 +3,6 @@
     public sealed class Settings
     {
         public string EnterspeedApiUri { get; set; } = "https://management.enterspeed.com/api/v1/";
```

Comment:
Minor thing, but maybe this should be renamed to ManagementApiUri now that we also have the IngestApiUri as both of them are enterpseed api uri's

Reply:
Good point, I'll change it.